datamodel.combodo-approval-extended.xml

```diff
@@ -551,7 +584,9 @@
 			return false;
 		}
 
-		$sAllowedProfiles = MetaModel::GetConfig()->GetModuleSetting('combodo-approval-extended', 'bypass_profiles', 'Administrator, Service Manager');
+	  $aAllowedClasses = MetaModel::GetConfig()->GetModuleSetting('combodo-approval-extended', 'targets', ApprovalConfiguration::DEFAULT_TARGET);
```

Comment:
```suggestion
		$aAllowedClasses = MetaModel::GetConfig()->GetModuleSetting('combodo-approval-extended', 'targets', ApprovalConfiguration::DEFAULT_TARGET);
```

Reply:
Please correct indentation.

Reply:
I found why my IDE was not formatting correctly. This time it should be ok.